doc/api/vm.md

```diff
@@ -436,6 +444,14 @@ changes:
     The `cachedDataProduced` value will be set to either `true` or `false`
     depending on whether code cache data is produced successfully.
     This option is deprecated in favor of `script.createCachedData()`.
+  * `importModuleDynamically` {Function} Called during evaluation of this
+    module when `import()` is called. This function has the signature
+    `(specifier, module)` where `specifier` is the specifier passed to
+    `import()` and `module` is this `vm.SourceTextModule`. The expected return
+    value is an object to be used as the namespace returned by `import()`. If
+    this option is not specified, calls to `import()` will reject with
+    [`ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING`][]. This method must return a
+    `vm.SourceTextModule`.
```

Comment:
Interesting, so the new changes will accept both a namespace or a vm.SourceTextModule instance as a return value? That seems like a good compromise to me that we then advocate that the module be returned as the "best practice" to ensure better spec-like behaviour.

Perhaps just note this in the wording, as it seems a little unclear currently that:
1. Both namespace objects and a vm.SourceTextModule object can be returned
2. That the vm.SourceTextModule is the recommended return value